lib/api/backoff.dart

```diff
@@ -66,8 +85,8 @@ class BackoffMachine {
   Future<void> wait() async {
     final bound = _minDuration(maxBound,
                                firstBound * pow(base, _waitsCompleted));
-    final duration = _maxDuration(const Duration(microseconds: 1),
-                                  bound * Random().nextDouble());
+    final duration = debugDuration ?? _maxDuration(const Duration(microseconds: 1),
+                                                   bound * Random().nextDouble());
```

Comment:
nit: line too long (and in particular the value of `microseconds` is past 80 columns, just)